sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -381,32 +383,33 @@ export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
           <FormHelperError>{formik.errors.siteColor}</FormHelperError>
         )}
       </FormRow>
-      {formik.dirty && (
-        <UnsavedChangesContainer>
-          <Box
-            css={{
-              position: 'absolute',
-              width: '100%',
-              height: '100%',
-              left: 0,
-              zIndex: -1,
-              backgroundColor: '$gray1',
-              opacity: 0.95,
-            }}
-          />
-          <Typography size="subheading" css={{ fontWeight: 600 }}>
-            You have unsaved changes
-          </Typography>
-          <Button
-            disabled={formik.isSubmitting}
-            type="submit"
-            size="md"
-            color="orange"
-          >
-            {formik.isSubmitting ? 'Saving...' : 'Save changes'}
-          </Button>
-        </UnsavedChangesContainer>
-      )}
+      {formik.dirty ||
+        (customLogo && (
```

Comment:
I think the correct logic here is
```suggestion
      {formik.dirty || customLogo)
         && (
```